test/integration/targets/ios_vlan/tests/cli/basic.yaml

```diff
@@ -0,0 +1,152 @@
+#  ios_config:
+#    lines:
+#      - switchport mode access
+#      - no switchport access vlan 100
+#    authorize: yes
+#    parents: "{{ item }}"
+#  with_items:
+#    - interface GigabitEthernet0/1
+#    - interface GigabitEthernet0/2
+
+#- name: create vlan
+#  ios_vlan: &create
+#    vlan_id: 100
+#    name: test-vlan
+#    authorize: yes
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == true"
+#      - "'vlan 100' in result.commands"
+#      - "'name test-vlan' in result.commands"
+
+#- name: create vlan(idempotence)
+#  ios_vlan: *create
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == false"
+
+#- name: Add interfaces to vlan
+#  ios_vlan: &interfaces
+#    vlan_id: 100
+#    interfaces:
+#      - GigabitEthernet0/1
+#      - GigabitEthernet0/2
+#    authorize: yes
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == true"
+#      - "'interface GigabitEthernet0/1' in result.commands"
+#      - "'switchport mode access' in result.commands"
+#      - "'switchport access vlan 100' in result.commands"
+#      - "'interface GigabitEthernet0/2' in result.commands"
+#      - "'switchport mode access' in result.commands"
+#      - "'switchport access vlan 100' in result.commands"
+
+#- name: Add interfaces to vlan(idempotence)
+#  ios_vlan: *interfaces
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == false"
+
+#- name: Remove interface from vlan
+#  ios_vlan: &single_int
+#    vlan_id: 100
+#    interfaces:
+#      - GigabitEthernet0/1
+#    authorize: yes
+#    register: result
+
+#- assert:
+#    that:
+#      - "result.changed == true"
+#      - "'vlan 100' in result.commands"
+#      - "'interface GigabitEthernet0/2' in result.commands"
+#      - "'switchport mode access' in result.commands"
+#      - "'no switchport access vlan 100' in result.commands"
+
+#- name: Remove interface from vlan(idempotence)
+#  ios_vlan: *single_int
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == false"
+
+#- name: Suspend vlan
+#  ios_vlan:
+#    vlan_id: 100
+#    state: suspend
+#    authorize: yes
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == true"
+#      - "'vlan 100' in result.commands"
+#      - "'state suspend' in result.commands"
+
+#- name: Unsuspend vlan
+#  ios_vlan:
+#    vlan_id: 100
+#    state: active
+#    authorize: yes
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == true"
+#      - "'vlan 100' in result.commands"
+#      - "'state active' in result.commands"
+
+#- name: delete vlan
+#  ios_vlan: &delete
+#    vlan_id: 100
+#    authorize: yes
+#    state: absent
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == true"
+#      - "'no vlan 100' in result.commands"
+
+#- name: delete vlan(idempotence)
+#  ios_vlan: *delete
+#  register: result
+
+#- assert:
+#    that:
+#      - "result.changed == false"
+
```

Comment:
Can add a test case for aggregate parameter

Reply:
Added the test in new commit.